README.md

```diff
@@ -56,16 +47,28 @@ _Note_: Feel free to use the `nightly` tag to test the latest version, i.e. `git
 ### Linux
 
 - [Download the latest release](https://github.com/gitpod-io/openvscode-server/releases/latest)
-- Untar and run the server:
-```bash
-tar -xzf openvscode-server-v${OPENVSCODE_SERVER_VERSION}.tar.gz
-cd openvscode-server-v${OPENVSCODE_SERVER_VERSION}
-./server.sh
-```
-- Visit the URL printed in your terminal. Just going to port 3000 won't work, because VS Code requires that you provide a uniquely generated security token to prevent unauthorized access.
+- Untar and run the server
+	```bash
+	tar -xzf openvscode-server-v${OPENVSCODE_SERVER_VERSION}.tar.gz
+	cd openvscode-server-v${OPENVSCODE_SERVER_VERSION}
+	./bin/openvscode-server # you can add arguments here, use --help to list all of the possible options
+	```
+
+  From the possible entrypoint arguments, the most notable ones are
+	- `--port` - the port number to start the server on, this is 3000 by default
+	- `--connection-token` & `--connection-secret` for securing access to the IDE, you can read more about it in [Securing access to your IDE](#securing-access-to-your-ide).
+	-  `--host` - determines the host the server is listening on. It defaults to `localhost`, so for accessing remotely it's a good idea to add `--host 0.0.0.0` to your launch arguments.
+
+- Visit the URL printed in your terminal.
 
 _Note_: You can use [pre-releases](https://github.com/gitpod-io/openvscode-server/releases) to test nightly changes.
 
+### Securing access to your IDE
+
+Since OpenVSCode Server v1.64, you can access the Web UI using a connection token, which prevents unauthorized access (by default anyone can access the IDE using just the hostname and port). If you start the server with `--connection-token YOUR_TOKEN`, the provided `YOUR_TOKEN` will be used and the authenticated URL will be displayed in your terminal once you start the server. If you want your development environment to be even more secure, create a plaintext file with the desired token as its contents and provide it to the server with `--connection-secret YOUR_SECRET_FILE`.
```

Comment:
```suggestion
Since OpenVSCode Server v1.64, you can access the Web UI without authentication (anyone can access the IDE using just the hostname and port), if you need some kind of basic authentication then you can start the server with `--connection-token YOUR_TOKEN`, the provided `YOUR_TOKEN` will be used and the authenticated URL will be displayed in your terminal once you start the server. You can also create a plaintext file with the desired token as its contents and provide it to the server with `--connection-secret YOUR_SECRET_FILE`.
```